code/WorkInProgress/AzrunStuff.dm

```diff
@@ -239,3 +239,353 @@
 			else if(prob(10))
 				boutput(src.owner,__red("You feel a sharp pain in your chest."))
 
+
+/datum/manufacture/ghost_brain // Move to manufacturing.dm
+	name = "Ghost Intelligence Core"
+	item_paths = list("MET-1","CON-1","ALL", "soulsteel")
+	item_amounts = list(6,5,3,5)
+	item_outputs = list(/obj/item/organ/brain/ghost)
+	time = 45 SECONDS
+	create = 1
+	category = "Component"
+
+/obj/item/organ/brain/ghost // Move to brain.dm
+	name = "Ghost Intelligence Core"
+	desc = "A brain shaped mass of silicon, soulsteel, and LED lights. Attempts to hold onto soul to give life to something else."
+	icon_state = "ghost_brain"
+	item_state = "ai_brain"
+	created_decal = /obj/decal/cleanable/oil
+	made_from = "pharosium"
+	var/activated = 0
+	var/lastTrigger
+	var/datum/movement_controller/ghost_brain/MC
+	var/obj/item/organ/brain/old_brain
+
+	New()
+		..()
+		MC = new
+
+	get_desc()
```

Comment:
It's not properly documented but get_desc actually receives two arguments: distance and user. You can use the latter instead of `usr`.